lua/lib/lfo.lua

```diff
@@ -15,6 +15,12 @@ local lfo_rates_as_strings = {"1/16","1/8","1/4","5/16","1/3","3/8","1/2","3/4",
 local params_per_entry = 14
 
 local rand_values;
+local percentage;
+local scaled_min, scaled_max, mid, centroid_mid;
+
+function LFO.init()
+  norns.lfo = {lattice = lattice:new()}
```

Comment:
seems this should check if norns.lfo exists prior to creating it again

Reply:
oh! i just assumed the `script.lua` clearout would always ensure it didn't exist, but def good to protect against weirdness. i'll adjust now!